src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -60,7 +60,61 @@ def test_WINNF_FT_S_SCS_4(self):
   def test_WINNF_FT_S_SCS_5(self):
     """New registration with TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 cipher.
 
-    Checks that SAS UUT response satisfy specific security conditions.
+    Checks that SAS UUT response satisfy cipher security conditions.
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_6(self):
+    """Unrecognized root of trust certificate presented during registration.
+    Checks that SAS UUT responds with fatal alert with unknown_ca.
+    """
+    device_cert = os.path.join('certs', 'unrecognized_device.cert')
+    device_key = os.path.join('certs', 'unrecognized_device.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key, alert_reason='tlsv1 alert unknown ca')
```

Comment:
As far as I can see, the test spec does not require us to provide a specific alert reason in this (or any other) test case.

Reply:
Kate, thank for reviewing. we will take it out. Other than this, are all other good for checking in ?

Reply:
Based on the code above, I'd expect the test code to (incorrectly) fail a SAS if it provides even a slightly different alert reason. Since the test spec does not require us to give any particular alert reason, I'd suggest removing line 210 of security_testcase.py and the input parameter alert_reason.

Unfortunately I don't personally have enough security knowledge to carefully review this PR. A number my coworkers are still out on vacation but I will ask them to review it when they get back (in about a week) if no one else has looked at it by then. If you can find another reviewer before then, that's probably most efficient.